frontend/e-buddy/src/app/auth/page.tsx

```
@@ -1,36 +1,60 @@
 
-const AuthPage = () => {
-  const { isAuthenticated } = useAuth();
+function FullPageLoader() {
+  return (
+    <div className="fixed inset-0 flex items-center justify-center bg-white z-50">
+      <svg
+        className="animate-spin h-10 w-10 text-gray-500"
+        xmlns="http://www.w3.org/2000/svg"
+        fill="none"
+        viewBox="0 0 24 24"
+      >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)